### PR TITLE
Implement option to ignore legacy verified status 

### PR DESCRIPTION
**Pull request by razvanazamfirei**
Introduced an option that lets users revoke legacy verified status for users subscribed to Twitter Blue. Currently, a legacy verified user will show a legacy verification badge if they are subscribed to Twitter Blue.

Note: The pop-up doesn't look great right now and I didn't want to mess with the structure too much as my experience with React is limited. Would appreciate suggestions/fixes!

### CODE REVIEW

File: src/_locales/es/messages.json [hunk 1]
```diff
@@ -23,6 +23,9 @@
     "hide_twitter_blue": {
         "message": "Ocultar íconos de Twitter Blue"
     },
+    "revoke_verified": {
+        "message": "Revocar las insignias verificadas de los suscriptores de Twitter Blue"
```
Comment (luiserdef) [inline src/_locales/es/messages.json:27]: In spanish could be: "Quitar ícono de verificado a usuarios con Twitter Blue."

**Review by luiserdef — CHANGES_REQUESTED** [review]
Hi!
Everything seems fine, just a minor suggestion for the translation in Spanish.
UI issues can be fixed. It's not a big problem, so don't worry about that.

File: src/_locales/es/messages.json (hunk 1 above)
Comment (razvanazamfirei) [reply src/_locales/es/messages.json]: Hi @luiserdef! Thank you for the suggestion! I made the change here and updated the english version as well for consistency.

**Comment by luiserdef** [comment]
Great, thanks for your constribution!

**Pull request merged**